### PR TITLE
Add locale example for lsNumber/Currency/EuroCurrency/TimeFormat

### PR DESCRIPTION
**Pull request by rscRoman**

### CODE REVIEW

File: data/en/lseurocurrencyformat.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
 	"name":"lsEuroCurrencyFormat",
 	"type":"function",
-	"syntax":"lsEuroCurrencyFormat(currency, type)",
+	"syntax":"lsEuroCurrencyFormat(currency [, type, locale])",
```
Comment (pfreitag) [inline data/en/lseurocurrencyformat.json]: It says that `type` is required so it should not be in square brackets here right?

Comment (rscRoman) [reply data/en/lseurocurrencyformat.json]: I'm not shure if type is required.
lsEuroCurrencyFormat(5) works with "currency" only

Comment (pfreitag) [reply data/en/lseurocurrencyformat.json]: Yes, according to the adobe docs `local` is the default type, so we should just update the `required` and `default` on the param below. Can you take care of that?

Comment (rscRoman) [reply data/en/lseurocurrencyformat.json]: Of course!
I do some updates and commit them.

**Comment by rscRoman** [comment]
@pfreitag 
Is there anything else needed on this Pull request?

**Comment by KamasamaK** [comment]
It looks like the given `values` for `locale` are just suggestions. That field is supposed list all values if enumerable. And I'm guessing `...` is invalid.

**Comment by rscRoman** [comment]
I am pretty sure that `...` is invalid.
Just to be sure that it is accepted for cfdocs, i took the values from [https://cfdocs.org/lsweek](url) as reference.

**Pull request merged**